builtin/logical/pki/crl_util.go

```diff
@@ -529,10 +550,22 @@ func buildCRLs(ctx context.Context, b *backend, req *logical.Request, forceNew b
 			}
 
 			var revokedCerts []pkix.RevokedCertificate
-			representative := issuersSet[0]
+			representative := issuerID("")
 			var crlIdentifier crlID
 			var crlIdIssuer issuerID
 			for _, issuerId := range issuersSet {
+				// Skip entries which aren't enabled for CRL signing. We don't
+				// particularly.
```

Comment:
"We don't particularly." <-- missing the rest of the sentence?

Reply:
Honestly? No clue! :-D 

Maybe "We don't particularly which issuer we ultimately choose otherwise?"